README.md

```diff
@@ -1,2 +1,43 @@
 # wpaustralia.org
-The WP Australia site http://wpaustralia.org/
+
+The WP Australia site [http://wpaustralia.org/]([http://wpaustralia.org/).
+
+### Installation
+
+We've created a script that automatically sets up a local development server using [Chassis](http://docs.chassis.io/en/latest/).
+
+You can get up and running by doing the following:
+1. Open Terminal.
+2. Change directories into the a folder where you would like to download the project. e.g. `cd /Volumes/Sites`
```

Comment:
`... directories into the folder`, rather than `... directories into the a folder` please